libs/martech/martech.js

```diff
@@ -143,9 +145,13 @@ const getTargetPersonalization = async () => {
   } else {
     sendTargetResponseAnalytics(false, responseStart, timeout);
     manifests = handleAlloyResponse(response.result);
+    propositions = response.result && response.result.propositions;
```

Comment:
In the case that something unexpected is returned for response.result.propositions like undefined or null, is it better that window._satellite.track on lined 255 is not called, or that it's called with an empty array? if it's the latter, small suggestion:
`propositions = response.result && response.result.propositions || [];`